packages/relay-compiler/core/GraphQLWatchmanClient.js

```diff
@@ -24,6 +25,14 @@ class GraphQLWatchmanClient {
 
   static isAvailable(): Promise<boolean> {
     return new Promise(resolve => {
+      try {
+        const checkWatchman = execSync('command -v watchman').toString().trim();
+        if (checkWatchman === '') {
+          return resolve(false);
+        }
+      } catch {
+        return resolve(false);
+      }
```

Comment:
TIL about `command` ! ✨ 

How about using the exit status, though? (And made it async too, although it probably matters very little overall.)

```js
const exec = require("util").promisify(require("child_process").exec)
undefined
> exec("command -v does-not-exist").then(() => console.log("Exists!")).catch(() => console.log("Does not exist!"))
Promise { … }
> Does not exist!
> exec("command -v ls").then(() => console.log("Exists!")).catch(() => console.log("Does not exist!"))
Promise { … }
> Exists!
```

Reply:
I just realized that this won't work on windows. I will add something else.